docs-source/content/samples/simple/tanzu-kubernetes-service/_index.md

```diff
@@ -7,7 +7,7 @@ description: "Sample for using the operator to set up a WLS cluster on the Tanzu
 
 
 This sample demonstrates how to use the Oracle [WebLogic Server Kubernetes Operator](/weblogic-kubernetes-operator/) (hereafter “the operator”) to set up a WebLogic Server (WLS) cluster on the Tanzu Kubernetes Grid (TKG).
-After performing the sample steps, your WLS domain runs on a TKG Kubernetes cluster instance and you can manage your WLS domain by using the WebLogic Server Administration Console.
+After performing the sample steps, your WLS domain with domain source type of Model In Image runs on an TKG Kubernetes cluster instance. Once the domain has been provisioned you can monitor it using the WebLogic Administration console.
```

Comment:
with domain source type of Model In Image -> with a Model in Image domain source type
runs on an TKG -> runs on a TKG
Once the domain has been provisioned -> After the domain has been provisioned, (comma)
WebLogic Administration console -> WebLogic Server Administration Console

Reply:
fixed

Reply:
Fixed